web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1713,6 +1713,8 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   @volatile var cometGetTimeout = 140000
 
+  @volatile var putAjaxFnsInContainerSession = false
```

Comment:
I generally assume Antonio's code is always better than my own. :)

That said, his would fail silently if you tried to do a bad thing (change after boot) wouldn't it?